sessions/nextjs-dynamic-routes/lotr-dynamic-routes/pages/books/the-fellowship-of-the-ring.js

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+import Link from "next/link";
+import books from "../../lib/books.js";
+
+export default function Fellowship() {
+  const currentBook = books[0];
+
+  return (
+    <>
+      <h2>{currentBook.title}</h2>
+      <p>{currentBook.description}</p>
+      <Image
+        src={currentBook.cover}
+        alt={`cover image of ${currentBook.title}`}
+        width={250}
+        height={419}
+      />
+      <Link href="/books">Back to overview</Link>
+    </>
+  );
+}
```

Comment:
rename file to `pages/volumes/….js`

```suggestion
import { volumes } from "../../lib/data.js";

export default function Fellowship() {
  const currentVolume = volumes.find(({slug}) => slug === "the-fellowship-of-the-ring");
  
  if (!currentVolume) {
    return null;
  }
  
  const { title, description, cover, books } = currentVolume;

  return (
    <>
      <Link href="/volumes">← All Volumes</Link>
      <h2>{title}</h2>
      <p>{description}</p>
      <Image
        src={cover}
        alt={`Cover image of ${title}`}
        width={140}
        height={230}
      />
    </>
  );
}
```